code/game/turfs/turf.dm

```diff
@@ -431,14 +437,21 @@
 		if(SS_READY(SSair))
 			SSair.mark_for_update(src)
 
+		if(istype(W, /turf/space) && W.loc.dynamic_lighting == 0)
+			var/image/I = image(icon = 'icons/mob/screen1.dmi', icon_state = "white")
+			I.plane = LIGHTING_PLANE_MASTER
+			I.blend_mode = BLEND_ADD
+			W.overlays += I
+
 		W.levelupdate()
 
 		. = W
 
-	recalc_atom_opacity()
+	//recalc_atom_opacity()
 	if (SSlighting && SSlighting.initialized)
-		lighting_overlay = old_lighting_overlay
+		//lighting_overlay = old_lighting_overlay
 		affecting_lights = old_affecting_lights
+		/*
```

Comment:
I'm not allowed to comment on the whole block because it's too long. But why not just delete this stuff?